### PR TITLE
fix: USAGE of Express for types express

### PR DESCRIPTION
**Pull request by kozzzz-07**
In ESModules, this is a default export, so it is an inappropriate USAGE. 
Fix to refer to `default`.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

### CODE REVIEW

File: types/passport-saml/passport-saml-tests.ts [hunk 1]
```diff
@@ -5,29 +5,29 @@ import MultiSamlStrategy = require('passport-saml/multiSamlStrategy');
...
-                // key removed, null if no key is removed
-            }
-        },
-        cert: fs.readFileSync('/path/to/cert.crt', 'UTF8')
-    },
-    (profile: {}, done: (err: Error | null, user: {}, info?: {}) => void) => {
-        const user = {};
-        done(null, user);
-    }
+	{
```
Comment (danger-public) [inline types/passport-saml/passport-saml-tests.ts:8]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## passport-saml ([<kbd>unpkg</kbd>](https://unpkg.com/browse/passport-saml@latest/))
was missing the following properties:
1. SAML
1. SAML
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9980228038aa969bdfeecc8a2d0ccb1ef21e4fdd
</p>

**Comment by peterblazejewicz** [comment]
@kozzzz-07 It does not look ExpressJS changed to ES6 modules yet (in the soruce code), where is that information coming from? Can you link the reference?
This is the latest one:
https://github.com/expressjs/express/tree/5.0.0-alpha.8

**Comment by kozzzz-07** [comment]
@peterblazejewicz 
In the `express' page, it is described in CommonJS, but in the `express/index.d.ts`, it is described how to use ESMocules.
I checked it in my environment.
If you import with ESMocules, it will be exported with default export.

I use a translator for most of my work...

File: types/express/index.d.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@
 /* =================== USAGE ===================
 
     import * as express from "express";
-    var app = express();
+    var app = express.default();
```
Comment (DanielRosenwasser) [inline types/express/index.d.ts:13]:
```suggestion
    import express = require("express");
    var app = express();
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

I think this is not a real difference, which one is used (both are correct), but I prefer to see CommonJS specific imports when using CJS modules.

@kozzzz-07 kudos for keeping trying 💪

**Pull request merged**